src/lib/components/forms/form/form-components/text-input/text-input.component.html

```diff
@@ -0,0 +1,21 @@
+<mat-form-field class="w-100" [tcTooltip]="tooltipText" [placement]="tooltipPlacement">
+    <mat-label>{{ name }}</mat-label>
+    <input
+        matInput
+        [type]="isPassword ? 'password' : 'text'"
+        [(ngModel)]="value"
+        [formControl]="control"
```

Comment:
Man kombiniert nicht ngModel (Template-driven) und formControl (reactive forms). Du solltest da auch eine Warnung in der Console im Browser kriegen..
Stattdessen kannst du folgendes machen um auf das von der NgModel-Directive erstellte control zuzugreifen.
```suggestion
        #control="ngModel"
```

Reply:
i see!